extern/sector-storage/ffiwrapper/types.go

```diff
@@ -34,11 +34,15 @@ type Storage interface {
 }
 
 type Verifier interface {
-	VerifySeal(proof2.SealVerifyInfo) (bool, error)
-	VerifyWinningPoSt(ctx context.Context, info proof2.WinningPoStVerifyInfo) (bool, error)
-	VerifyWindowPoSt(ctx context.Context, info proof2.WindowPoStVerifyInfo) (bool, error)
+	VerifySeal(proof5.SealVerifyInfo) (bool, error)
+	VerifyAggregateSeals(aggregate proof5.AggregateSealVerifyProofAndInfos) (bool, error)
+	VerifyWinningPoSt(ctx context.Context, info proof5.WinningPoStVerifyInfo) (bool, error)
+	VerifyWindowPoSt(ctx context.Context, info proof5.WindowPoStVerifyInfo) (bool, error)
 
 	GenerateWinningPoStSectorChallenge(context.Context, abi.RegisteredPoStProof, abi.ActorID, abi.PoStRandomness, uint64) ([]uint64, error)
+
+	// cheap, makes no sense to put this on the storage interface
```

Comment:
Really, neither this nor `GenerateWinningPoStSectorChallenge` belong here. Why do this?

Reply:
Yeah, we probably want a separate interface for these. This is here because the Sealer instance is wrapped into a Worker+SealingMgr, and it really makes no sense to put those 2 calls behind those layers of abstraction.

The `Verifier` interface is the other ready-to-use interface that's already plumbed into all the places we need those calls in.